.ci/build-linux.sh

```diff
@@ -44,6 +44,7 @@ cmake ..                                               \
     -DUSE_SDL=ON                                       \
     -DUSE_SYSTEM_FFMPEG=OFF                            \
     -DUSE_DISCORD_RPC=ON                               \
+    -DQT_USE_NODISCARD_FILE_OPEN \
```

Comment:
Parse error in command line argument: QT_USE_NODISCARD_FILE_OPEN
Should be: VAR:type=value

Reply:
I'll just remove the commit.
It's not really relevant